src/views/Receiver.vue

```diff
@@ -80,6 +80,8 @@ video
 
 #volume-slider
   max-width: 120px
+  @supports (-webkit-touch-callout: none) // iOS volume slider doesn't work, so hide it
```

Comment:
This should be in a different PR / ticket... was this written in this PR or did it end up in this branch after pulling from master?